package.json

```diff
@@ -8,8 +8,8 @@
 		"type": "git",
 		"url": "https://github.com/camel-tooling/vscode-camel-extension-pack.git"
 	},
-	"preview": true,
-	"version": "0.0.14",
+	"preview": false,
```

Comment:
I'm wondering if we can remove the line for preview. I suppose that when not set, the behavior is to not consider the extension as a preview

Reply:
According to the anouncement it should be enough with removing it.

I can't test it `vsce package` because all created with that are marked with preview in my vscode. I took a quick peek to the command --help and did a quick search but I didn't found anything on how to avoid that. I don't think is worth the time looking how to package in no-preview mode as it should work, though.